### PR TITLE
Basic 2 Camera Movement

### PR DESCRIPTION
**Pull request by kev123361**
Functionality is only present in cameratest scene.

### CODE REVIEW

File: Assets/Scripts/Camera/CameraFollow.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    // Use this for initialization
+    void Start () 
+    {
+        currentz = transform.position.z;
+    }
+    
+    // LateUpdate is called after Update each frame
+    void LateUpdate () 
+    {
+        Vector2 targetPos = Vector2.SmoothDamp(transform.position, player.transform.position, ref velocity, Time.deltaTime * followSpeed);
```
Comment (jpbronsted) [inline Assets/Scripts/Camera/CameraFollow.cs]: Not sure if we care too much about style here, but if we do, let's try to keep lines from being too long. I tend to advocate for the 80 column maximum, but I'm open to discussion.

**Comment by jpbronsted** [comment]
Is this pull request related to one of the open issues?

File: Assets/Scripts/Camera/CameraFollow.cs (hunk 1 above)
Comment (kev123361) [reply Assets/Scripts/Camera/CameraFollow.cs]: Ah, yeah we can definitely do that. I always just forget cus I work on a pretty wide screen, but I'll set up my visual studio for that.

**Comment by kev123361** [comment]
> Is this pull request related to one of the open issues?

Yeah, I opened up the issue before I started, just couldn't figure out how to attach the PR to the issue.

**Comment by jpbronsted** [comment]
> > Is this pull request related to one of the open issues?
> 
> Yeah, I opened up the issue before I started, just couldn't figure out how to attach the PR to the issue.

You can just leave the issue number in a comment, like so: #2. You can also put it in the commit text for one or any of the commits that the PR encapsulates (i.e. `git commit -m "resolves #2"`).

**Comment by kev123361** [comment]
> > > Is this pull request related to one of the open issues?
> > 
> > 
> > Yeah, I opened up the issue before I started, just couldn't figure out how to attach the PR to the issue.
> 
> You can just leave the issue number in a comment, like so: #2. You can also put it in the commit text for one or any of the commits that the PR encapsulates (i.e. `git commit -m "resolves #2"`).

gotcha, I was looking for it in the sidebar where you can assign milestones and stuff like that. I'll do it for the future

**Pull request merged**